Application/GBAppledocApplication.m

```diff
@@ -269,6 +269,7 @@ - (void)application:(DDCliApplication *)app willParseOptions:(DDGetoptLongParser
         { kGBArgDocSetXMLFilename,                                          0,      DDGetoptRequiredArgument },
 		{ kGBArgDocSetPackageFilename,										0,		DDGetoptRequiredArgument },
 		
+        { "no-covert-pain-text-html", 0, DDGetoptNoArgument},
```

Comment:
Is it too late to point out this?
`no-covert-pain-text-html`  should be
`no-convert-pain-text-html`